File: 18406.cpp
#include <iostream>
using namespace std;

int main() {
	string s; cin >> s;

	int left = 0;
	for (int i = 0; i < s.length() / 2; i++)
		left += s[i] - '0';

	int right = 0;
	for (int i = s.length() / 2; i < s.length(); i++)
		right += s[i] - '0';

	if (left == right) cout << "LUCKY\n";
	else cout << "READY\n";
}

File: 18406.py
s = input()

left = 0
for i in range(len(s) // 2):
    left += int(s[i])

right = 0
for i in range(len(s) // 2, len(s)):
    right += int(s[i])

if left == right: print("LUCKY")
else: print("READY")
